app/service/form_update_service.rb

```diff
@@ -0,0 +1,23 @@
+class FormUpdateService
+  class << self
+    def save(form)
+      form.save
+    end
+
+    def update(form, **options)
+      form.update(**options)
+    end
+
+    def destroy!(form)
+      form.destroy!
+    end
```

Comment:
not sure I'm following this service. It feels like we just trying to duplicate what ActiveRecord already gives us.

Reply:
@thomasiles @aliuk2012 and I discussed and agreed a different approach, see [64290fa](https://github.com/alphagov/forms-api/pull/232/commits/64290fa4f14a8225abe7319025a7eaddc68682a5).